### PR TITLE
Improvements to automatic deactivation of admin accounts

### PR DESCRIPTION
**Pull request by gauthierm**
https://trello.com/c/57xnmGP8/632-0-5d-add-error-message-to-failed-logins-that-failed-because-of-90-days-of-inactivity

https://trello.com/c/kJ5GgOP9/633-0-5d-add-reset-method-for-inactive-accounts-to-allow-them-to-be-active-again

 * show a message when you try to sign in with an inactive account
 * show inactive accounts on index page

### CODE REVIEW

File: Admin/dataobjects/AdminUser.php [hunk 1]
```diff
@@ -171,8 +184,8 @@ public function isAuthenticated(AdminApplication $app)
 			$authenticated = true;
 		}
 
-		if ($this->force_change_password)
-			$authenticated = false;
+		$authenticated = ($authenticated && $this->isActive());
+		$authenticated = ($authenticated && !$this->force_change_password);
```
Comment (isaacgrant) [inline Admin/dataobjects/AdminUser.php]: Any reason this isn't all one statement?

File: Admin/components/AdminUser/Index.php [hunk 2]
```diff
@@ -88,6 +97,26 @@ protected function buildInternal()
 
 		$date_renderer = $date_column->getRendererByPosition();
 		$date_renderer->display_time_zone = $this->app->default_time_zone;
+
+		$this->buildActiveNote();
```
Comment (isaacgrant) [inline Admin/components/AdminUser/Index.php]: Should this message only show up if there is inactive questions on the page?

**Comment by isaacgrant** [comment]
R+ beyond minor feedback above. Nice cleanup.

File: Admin/dataobjects/AdminUser.php (hunk 1 above)
Comment (gauthierm) [inline Admin/dataobjects/AdminUser.php]: Ease of readability. I agree it could be one statement and still be easy to read.

**Comment by gauthierm** [comment]
Feedback addressed.

**Pull request merged**